subsys/net/lib/aws_jobs/src/aws_jobs.c

```diff
@@ -464,6 +465,7 @@ int aws_jobs_subscribe_job_id_get_topic(struct mqtt_client *const client,
 	struct mqtt_topic subscribe_topic;
 	int err = construct_job_id_get_topic(client->client_id.utf8,
 					     job_id,
+					     "/#",
```

Comment:
I would appreciate some comment on why `/#` is used as a suffix here. Up to you to decide if it makes sense :)

Reply:
I'll add it it's MQTT speak for `/*` on a topic category